exchange/index_test.go

```diff
@@ -658,12 +659,12 @@ func TestCleanBlockStore(t *testing.T) {
 	idx, err := NewIndex(ds, bs)
 	require.NoError(t, err)
 
-	// generate random block1
+	// generate random block1 (codec: dagcbor)
 	blk1 := testutil.CreateRandomBlock(t, idx.Bstore())
 	require.NoError(t, idx.Bstore().Put(blk1))
 
-	// generate random block2
-	blk2 := testutil.CreateRandomBlock(t, idx.Bstore())
+	// generate random block2 (codec: raw)
+	blk2 := gstestutil.GenerateBlocksOfSize(1, 10000)[0]
```

Comment:
@tchardin GenerateBlocksOfSize() generates Raw blocks, I've tested it on the main branch and it fails but it does work on this branch